web-components/tsconfig.json

```diff
@@ -5,22 +5,17 @@
     "declaration": true,
     "jsx": "react-jsx",
     "noEmitOnError": true,
+    "types": ["node", "jest"],
     "outDir": "lib",
     "rootDir": "src",
     "module": "esnext",
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
+    "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
     "allowSyntheticDefaultImports": true,
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "noEmit": true
+    "target": "es6"
```

Comment:
also might be able to remove this - it shoudl be inherited from the `tsconfig` file in teh app root, but didn’t seem to be working correctly